index.html

```diff
@@ -1,21 +1,28 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <link rel="icon" href="/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Vite App</title>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/public/academia.png" />
```

Comment:
Nesses casos, o caminho já é para public por padrão, não tem necessidade do /public
```suggestion
    <link rel="icon" href="/academia.png" />
```